docs/contributing/contributing-rules.md

```diff
@@ -102,14 +102,15 @@ In addition to the fields mentioned above, rules submitted to Semgrep Registry h
         </tr>
         <tr>
             <td>
-              Additionally required by <code>category: security</code>:
+              Additional keys required when <code>category</code> is <code>security</code>:
```

Comment:
![Snag_9f9f67](https://github.com/user-attachments/assets/f4072bd8-71e3-450f-ae2f-e58329c0bfa5)
The white boxes here just look a little off? Maybe try `<pre>` vs `<code>`? Otherwise it may be a css issue - let me know and I can try to fix in a separate PR.

Reply:
Fixed it! 
<img width="448" alt="image" src="https://github.com/user-attachments/assets/50cb4d77-dfcb-422c-a90b-5b13958c7bf3">